### PR TITLE
File Added : Assignment 5 & 6

### PR DESCRIPTION
**Pull request by roshnidurugwar**

### CODE REVIEW

**Review by Oshin-250197 — APPROVED** [review]

**Review by arshad05786 — APPROVED** [review]

**Review by SamikshaWani — APPROVED** [review]

File: Roshni/html/Assignment5/Ass5-Q1.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                        <h2 class="right"><i>About Me&nbsp;&nbsp;</i></h2>
+                        <div class="shadow" style="background-image:url('./image1.jpg');background-position: center;
+                        background-repeat: no-repeat;
+                        background-size: cover;width:96%;padding:7px;">
+                                <br>
+                                I am Roshni Durugwar<br>
+                                I am From Orange City<br>
+                                I am Student of Ambersoft Acedemy<br>
+                                My College name is S.B.Jain Institute Of Technology,Nagpur<br>
+
```
Comment (Priyankashelke2411) [inline Roshni/html/Assignment5/Ass5-Q1.html]: remove extra line

File: Roshni/html/Assignment5/Photo.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        <title>Photo Gallary</title>
+    </head>
+    <body bgcolor="pink">
+        <style>
+            a:link {
+                color: green;
+                text-decoration: none;
+                text-align: center;
+            }
+
```
Comment (Priyankashelke2411) [inline Roshni/html/Assignment5/Photo.html]: remove extra line

**Review by Priyankashelke2411 — CHANGES_REQUESTED** [review]

File: Roshni/html/Assignment5/Ass5-Q1.html (hunk 1 above)
Comment (roshnidurugwar) [reply Roshni/html/Assignment5/Ass5-Q1.html]: done maam

File: Roshni/html/Assignment5/Photo.html (hunk 2 above)
Comment (roshnidurugwar) [reply Roshni/html/Assignment5/Photo.html]: done maam

File: Roshni/html/Assignment5/Ass5-Q1.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    <head>
+        <title>Assignment 5</title>
+        <meta name="description" content="Home">
+        <meta name="keywords" content="">
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=1" />
+        <link rel="stylesheet" href="Ass5-Q1.css"/>
+    </head>
+    <body>
+            <table id="table1">
```
Comment (KomalShinde813) [inline Roshni/html/Assignment5/Ass5-Q1.html]: check spaces may be 8 spaces are there before table tag

File: Roshni/html/Assignment5/Ass5-Q1.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+                                My College name is S.B.Jain Institute Of Technology,Nagpur<br>
+                         </div>
+                        <div>
+                            <br><br><br><br><br>
+                            <h2 class="right"><i>More About Me&nbsp;&nbsp;</i></h2>
+                        </div>
+                        <div>
+                            <div class="shadow" style="background-image:url('./image1.jpg');background-position: center;
+                            background-repeat: no-repeat;
+                            background-size: cover;width:96%;padding:7px;">
```
Comment (KomalShinde813) [inline Roshni/html/Assignment5/Ass5-Q1.html]: space required after ; and : in style

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

File: Roshni/html/Assignment5/Ass5-Q1.html (hunk 3 above)
Comment (roshnidurugwar) [reply Roshni/html/Assignment5/Ass5-Q1.html]: done maam

File: Roshni/html/Assignment5/Ass5-Q1.html (hunk 4 above)
Comment (roshnidurugwar) [reply Roshni/html/Assignment5/Ass5-Q1.html]: done maam

File: Roshni/html/Assignment5/Ass5-Q1.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+                    <div class="shadow" style="background-image: url('./image1.jpg'); background-position: center;
+                    background-repeat: no-repeat;
+                    background-size: cover; width: 96%; padding: 7px;">
+                        <br>
+                        I am Roshni Durugwar<br>
+                        I am From Orange City<br>
+                        I am Student of Ambersoft Acedemy<br>
+                        My College name is S.B.Jain Institute Of Technology,Nagpur<br>
+                    </div>
+                        <div>
```
Comment (KomalShinde813) [inline Roshni/html/Assignment5/Ass5-Q1.html]: check indentation here

File: Roshni/html/Assignment5/Ass5-Q1.html [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+                            <br><br><br><br><br>
+                            <h2 class="right"><i>More About Me&nbsp;&nbsp;</i></h2>
+                        </div>
+                        <div>
+                            <div class="shadow" style="background-image: url('./image1.jpg'); background-position: center;
+                            background-repeat: no-repeat;
+                            background-size: cover; width: 96%; padding: 7px;">
+                            <ul>
```
Comment (KomalShinde813) [inline Roshni/html/Assignment5/Ass5-Q1.html]: check indentation use prettier

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

File: Roshni/html/Assignment5/Ass5-Q1.html (hunk 6 above)
Comment (roshnidurugwar) [reply Roshni/html/Assignment5/Ass5-Q1.html]: yes maam

File: Roshni/html/Assignment5/Ass5-Q1.html (hunk 5 above)
Comment (roshnidurugwar) [reply Roshni/html/Assignment5/Ass5-Q1.html]: Yes ma'am I did it

**Review by KomalShinde813 — APPROVED** [review]

**Pull request merged**